competitions/cloud-cover/benchmark-tutorial.ipynb

```diff
@@ -0,0 +1,2516 @@
+    "\n",
+    "**In this challenge, your goal is to detect cloud cover in satellite imagery**. \n",
+    "\n",
+    "The challenge uses publicly available satellite data from the [Sentinel-2 mission](https://sentinel.esa.int/web/sentinel/missions/sentinel-2), which captures wide-swath, high-resolution, multi-spectral imaging. For each tile, data is separated into 13 spectral bands capturing the full visible spectrum, near-infrared, and infrared light. Feature data is shared through Microsoft's [Planetary Computer](https://planetarycomputer.microsoft.com/).\n",
+    "\n",
+    "The ground truth labels for cloud cover are part of a new dataset created by [Radiant Earth Foundation](https://www.radiant.earth/). After the competition ends, the labels dataset will be made publicly available to contribute to ongoing learning and advancement in the field.\n",
+    "\n",
+    "In this post, we'll cover:\n",
+    "\n",
+    "- [Exploring the data](#explore-the-data)\n",
+    "- [Preparing the data](#split-data)\n",
+    "- [Following the code execution submission format](#code-exec)\n",
+    "- [Building the benchmark model](#build-model)\n",
+    "- [Fitting the benchmark model](#fit-model)\n",
+    "- [Generating a code execution submission](#generate-submission)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%%capture\n",
+    "!pip install pandas_path pytorch_lightning cloudpathlib loguru typer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import shutil\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from pandas_path import path\n",
+    "from pathlib import Path\n",
+    "from PIL import Image\n",
+    "import pytorch_lightning as pl\n",
+    "import torch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "<a id=\"explore-the-data\"></a>\n",
+    "\n",
+    "## Explore the data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we'll download all of the competition data to a local folder. For instructions, see the competition's data download [page](https://www.drivendata.org/competitions/83/cloud-cover/data/). You'll need to join the competition in order to view the data page. \n",
```

Comment:
Does this need to be updated to mention that the training data are available on the Planetary Computer at `/driven-data/cloud-cover`.